lips/lip-8.md

```diff
@@ -17,14 +17,22 @@ An extension of the Off-Chain protocol to provide support for more advanced merc
 
 Version 0 of the Off-Chain Protocol is described in [LIP 1](https://lip.libra.org/lip-1/).  Version 1 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
 
+---
+# Disclaimer
+---
+
+This LIP does not contain the initial phase of a sub-account discovery that is required to start negotiating merchant scenarios.
+The process described below starts at the phase that both sides (particularly the biller side) have the relevant subaddresses. 
+For now, we will assume that such a discovery process took place and the merchant have the buyer subaddress at hand.
+  
 ---
 # Specification
 ---
 
 # Fund Pull Pre-Approval
 ### *VERSION SUPPORT: Supported only in version 1 of off-chain APIs*
 
-Establishes a relationship between sender and recipient where the recipient can then pull funds from the sender without sender approving each transaction.  This allows recipient to bill the sender without sender approving each payment.  This relationship exists between a subaddress on the biller side and a subaddress on the sender side.  After this request is POSTed, the target VASP can use out-of-band methods to determine if this request should be granted.  If the target VASP chooses to allow the relationship to be established, the biller can create a payment object and POST to the billed party’s VASP to request funds.  The “funds_pull_approval_id” object must then match the ID established by this request.
+Establishes a relationship between the sender and recipient where the recipient can now pull funds from the sender without the sender approving each transaction.  This allows the recipient to bill the sender without the sender approving each payment.  This relationship exists between a subaddress on the biller side and a subaddress on the sender's side.  After this request is POSTed, the target VASP can use out-of-band methods to determine if this request should be granted.  If the target VASP chooses to allow the relationship to be established, the biller can create a payment object and POST to the billed party’s VASP to request funds.  The “funds_pull_approval_id” object must then match the ID established by this request.
```

Comment:
Here's another assumption, an out-of-band process, I think having a strawman definition of this protocol broken down into bullets would be more accessible. We could even allude to one such method for doing the exchanges.